src/appendices/hed.md

```diff
@@ -165,38 +167,55 @@ field of the `dataset_description.json` file located in the dataset root directo
 This allows for a proper validation of the HED annotations
 (for example using the `bids-validator`).
 
-Example: The following `dataset_description.json` file specifies that the
-[`HED8.1.0.xml`](https://github.com/hed-standard/hed-schemas/blob/main/standard_schema/hedxml/HED8.1.0.xml)
+**Example:** The following `dataset_description.json` file specifies that the
+[`HED8.2.0.xml`](https://github.com/hed-standard/hed-schemas/blob/main/standard_schema/hedxml/HED8.2.0.xml)
 file from the `standard_schema/hedxml` directory of the
 [`hed-schemas`](https://github.com/hed-standard/hed-schemas)
 repository on GitHub should be used to validate the study event annotations.
 
 ```JSON
 {
   "Name": "A great experiment",
-  "BIDSVersion": "1.7.0",
-  "HEDVersion": "8.1.0"
+  "BIDSVersion": "1.8.0",
+  "HEDVersion": "8.2.0"
 }
 ```
 
-If you omit the `HEDVersion` field from the dataset description file,
-a warning will be generated and
-any present HED information will be validated using the latest version of the HED schema.
-This is bound to result in problems, and hence, it is strongly RECOMMENDED that the
-`HEDVersion` field be included when using HED in a BIDS dataset.
+An error will be generated by the BIDS validator if
+your dataset uses HED and the `HEDVersion`
+field is missing from the dataset description file. To avoid this,
+include a `HEDVersion` field in the dataset_description.json
+if you are using HED annotations.
 
 ### Using HED library schemas
 
 HED also allows you to use one or more specialized vocabularies along with or instead of
 the standard vocabulary. These specialized vocabularies are developed by
 communities of users and are available in the GitHub
 [hed-schemas](https://github.com/hed-standard/hed-schemas) repository.
-Library schema are specified in the form `<library-name<_>library-version>`.
```

Comment:
Text below should be "A library schema is..."

Reply:
Changed [here](https://github.com/bids-standard/bids-specification/pull/1592/commits/5c0190cd3e7211717b9d099aa209c1c9d2be134b).